### DISCUSSION TITLE
Running only VCF file

### DISCUSSION THREAD

**Discussion by mateusfalco** [exomiser/Exomiser#551] [General]
Hi,
I'm facing a trouble cause a need to run only a vcf file without parameter --sample phenopacket.yml.
There is a way to do that, run only with --vcf parameter?

**Comment by julesjacobsen** [comment]
What are you trying to achieve? Exomiser is designed to run a phenotype-based variant prioritisation, so if you're not using the phenotype, you're not going to get good results.

**Reply by mateusfalco** [reply]
Right! I get your point, but if a have a sample that don't have any phenotype to add, how can i proceed with that, there is a way or i need to run another way?

So, i get what i'm looking..

Thanks Jules for your time and considerations
